packages/launchpad/src/setup/TestingTypeCards.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+  <div class="max-w-4xl mx-auto text-center">
+    <TestingTypeCard
+      v-if="ct.isSetup"
+      :id="ct.id"
+      :title="ct.title"
+      description="LAUNCH"
+      role="launch-component-testing"
+      @click="emit('launchCt')"
+    />
+
+    <TestingTypeCard
+      v-else
+      :id="ct.id"
+      :title="ct.title"
+      :description="ct.description"
+      role="setup-component-testing"
+      @click="selectTestingType('component')"
+    />
+
+    <TestingTypeCard
+      v-if="e2e.isSetup"
+      :id="e2e.id"
+      :title="e2e.title"
+      description="LAUNCH"
+      role="launch-e2e-testing"
+      @click="emit('launchE2E')"
+    />
+
+    <TestingTypeCard
+      v-else
+      :id="e2e.id"
+      :title="e2e.title"
+      :description="e2e.description"
+      role="setup-e2e-testing"
+      @click="selectTestingType('e2e')"
+    />
+  </div>
+</template>
+
+<script setup lang="ts">
+import { gql } from "@urql/core";
+import { useMutation } from "@urql/vue";
+import { computed } from "vue";
+import { 
+  TestingTypeAppQueryFragment, 
+  TestingTypeWizardQueryFragment, 
+  TestingTypeEnum,
+  TestingTypeSelectDocument 
+} from "../generated/graphql";
+import TestingTypeCard from "./TestingTypeCard.vue";
+
+gql`
+fragment TestingTypeCardsApp on App {
+  activeProject {
+    hasSetupComponentTesting
+    hasSetupE2ETesting
+  }
+}
+`
+  
+gql`
+fragment TestingTypeCardsWizard on Wizard {
```

Comment:
Any reason these two couldn't be consolidated into:

```graphql
fragment TestingTypeCards on Query {
  app {
    activeProject {
      hasSetupComponentTesting
      hasSetupE2ETesting
    }
  }
  wizard {
    testingTypes {
      id
      title
      description
    }
  }
}
```

Reply:
I could not get `mountFragment` to work with Fragments on the top level `Query`. I wonder if I'm missing something.